packages/bun-types/fs.d.ts

```diff
@@ -176,6 +176,17 @@ declare module "fs" {
      * @since v0.0.67
      */
     name: string;
+    /**
+     * The base path that this `fs.Dirent` object refers to.
+     * Deprecated in favor of `parentPath`.
+     * @since v??????
```

Comment:
this should be removed

Reply:
Seems like this entire file gets deleted to resolve the merge conflict -- if I'm understanding things correctly, type definitions are now in https://github.com/DefinitelyTyped/DefinitelyTyped per #7670.